src/main/scala/za/co/absa/hyperdrive/trigger/Application.scala

```diff
@@ -85,6 +85,7 @@ object ObjectMapperSingleton {
     .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
     .configure(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES, false)
     .configure(DeserializationFeature.FAIL_ON_NULL_CREATOR_PROPERTIES, true)
+    .configure(DeserializationFeature.USE_LONG_FOR_INTS, true)
```

Comment:
I think we've already discussed this "bug" and I went for `@JsonDeserialize` annotation see our whatsapp and see `JobsToRun` class.

I am not sure, right now its fine and I tried all endpoints and it's working fine

Reply:
Ah yes I didn't remember. Thanks for pointing it out. Will change it to be consistent